test/src/main/scala/scala/test/twitter_scrooge/BUILD

```diff
@@ -0,0 +1,100 @@
+load("//scala:scala.bzl",   "scala_binary", "scala_library")
+load("//twitter_scrooge:twitter_scrooge.bzl", "scrooge_scala_library")
+
+scrooge_scala_library(
+  name = "thrift1",
```

Comment:
can we call this `scrooge1` or something, it is a bit confusing with thrift libraries and scrooge libraries.